paper-shadow.html

```diff
@@ -23,34 +23,90 @@
 -->
 
 <link href="../polymer/polymer.html" rel="import">
+<link href="../layout/layout.html" rel="import">
+<dom-module id="paper-shadow">
+  <style>
+    paper-shadow {
+      display: block;
+      position: relative;
+      pointer-events: none;
+    }
+
+    paper-shadow #shadow-bottom,
+    paper-shadow #shadow-top {
+      border-radius: inherit;
+    }
+
+    paper-shadow #shadow-bottom[animated],
+    paper-shadow #shadow-top[animated] {
+      transition: box-shadow 0.28s cubic-bezier(0.4, 0, 0.2, 1);
+    }
+
+    paper-shadow #shadow-top.paper-shadow-z-1 {
+      box-shadow: none;
+    }
+
+    paper-shadow #shadow-bottom.paper-shadow-z-1 {
+      box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.37);
+    }
```

Comment:
Using attributes still modifies the host element, which still doesn't seem like a good idea even if you try to use a more obscure attribute than `class`?

Maybe this whole element is more appropriate as shared styles. Instead of having a `<paper-shadow>` as a child of your element, you include some common Polymer/paper-styles repo and add the "paper-shadow" class.